build.gradle

```diff
@@ -54,6 +54,12 @@ buildscript {
     classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
     classpath "com.jakewharton:butterknife-gradle-plugin:$butterknife_version"
 
+
+    classpath "org.sagebionetworks.research:domain:$sage_research_version"
+    classpath "org.sagebionetworks.research:presentation:$sage_research_version"
+    classpath "org.sagebionetworks.research:data:$sage_research_version"
```

Comment:
Shouldn't need to add these here. The sub modules that need sage research depend on them directly.

Reply:
Fixed